js/bundle.go

```diff
@@ -217,6 +218,18 @@ func (b *Bundle) Instantiate() (bi *BundleInstance, instErr error) {
 		jsOptionsObj = jsOptions.ToObject(rt)
 	}
 	b.Options.ForEachSpecified("json", func(key string, val interface{}) {
+		switch v := val.(type) {
+		case null.Bool:
+			val = v.ValueOrZero()
+		case null.Float:
+			val = v.ValueOrZero()
+		case null.Int:
+			val = v.ValueOrZero()
+		case null.String:
+			val = v.ValueOrZero()
+		case null.Time:
+			val = v.ValueOrZero()
```

Comment:
The repetition here is silly to me, but I didn't find a way to make it more generic (without using `reflect`). Is there a cleaner / more concise way of doing this?

Reply:
You can use go interfaces to check for all types that have a particular method, `ValueOrZero()` in this case. Add something like:
```go
type valueOrZeroer interface {
  ValueOrZero() time.Time
}
```
then you can check if the type is `valueOrZeroer` in the type switch

Reply:
Nevermind, the previous response didn't take into account the different return types

Reply:
@na-- Sorry, so you're saying that this can't be DRY'd? :-/

I wish the switch with type assignment worked for `case null.Bool, null.Int, null.String, ...` so there could only be one `val = v.ValueOrZero()` expression, but I get why it doesn't. The repetition here is killing me, though... :)

Reply:
@na-- Take a look at latest commit 0da876c6 (I'll squash it before merging).

Unfortunately, this pretty much moved where the `switch` happens, since we need to properly convert `reflect.Value`s, but I think I agree with you that it's cleaner here as it already deals with reflection and type checking.

It's also handling `types.NullDuration` properly now.